WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -406,6 +404,28 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     }
 }
 
+// MARK: - Private helpers
+private extension AuthenticationManager {
+    func checkJetpackErrorForSelfHostedSite(url: String) -> Bool {
```

Comment:
Super nit:  Assuming that this method doesn't have to know about what `JetpackError` presentation is, would a name like `isJetpackInstallationValidForSelfHostedSite` read better?